languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -20251,6 +20280,20 @@ though, and has since been slightly modified)-->
       <example correction="De toute façon"><marker>De toutes façons</marker></example>
       <example><marker>De toute façon</marker></example>
     </rule>
+      <rule id="A_TOUTE_A_L_HEURE" name="à tout à l'heure">
+          <pattern>
+              <token>à</token>
+              <marker>
+                  <token>toute</token>
+              </marker>
+              <token>à</token>
+              <token>l'</token>
+              <token>heure</token>
+          </pattern>
+          <message>Voulez-vous écrire <suggestion>tout</suggestion> ?</message>
+          <url>https://www.lalanguefrancaise.com/orthographe/a-tout-a-l-heure-ou-a-toute-a-l-heure/</url>
+          <example correction="tout">À <marker>toute</marker> à l'heure.</example>
+      </rule>
```

Comment:
With the last commit, this old version of the rule somehow found its way back into the file.

Before I merge this, can you please delete this part? It is identical with TOUTE_A_L_HEURE[1].

Also, I recommend not `temp_off`ing the single `<rule>`s, but the entire `<rulegroup>`.